flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/sink/AbstractStreamingCommitterHandler.java

```diff
@@ -49,10 +49,10 @@
  * @param <InputT> The input type of the {@link Committer}.
  * @param <CommT> The committable type of the {@link Committer}.
  */
-abstract class AbstractStreamingCommitterOperator<InputT, CommT>
-        extends AbstractStreamOperator<CommT> implements OneInputStreamOperator<InputT, CommT> {
-
-    private static final long serialVersionUID = 1L;
```

Comment:
Is it safe to remove this line?

Reply:
You mean `serialVersionUID`? With `StreamOperatorFactory` (which are serializable) there is actually no need for operators to be serializable anymore and it will be removed in the long run.

Reply:
Note that factories should have `serialVersionUID` if we are very strict. However, actually in non-persistent serialization none of it actually matters. `serialVersionUID` is only important if you need a way to read old serialized data into changed serializable classes.